docs/contents.md

```diff
@@ -1,33 +1,33 @@
 
 # Contents
 
 The **Phishing Email Response** solution pack contains the following resources.
 
 ## Global Variable
 
-|Name|Description|
-| :- | :- |
-|`Default_Email`| This global variable contains the email address which sends acknowledgement and other emails to the reporter. |
+| Name            | Description                                                                                                   |
+|:----------------|:--------------------------------------------------------------------------------------------------------------|
+| `Default_Email` | This global variable contains the email address which sends acknowledgement and other emails to the reporter. |
 
 ## Record Set
 
-|Name|Description|
-| :- | :- |
-|Scenario| A simulation that helps you understand the *Phishing Email Response* solution pack.|
+| Name     | Description                                                                         |
+|:---------|:------------------------------------------------------------------------------------|
+| Scenario | A simulation that helps you understand the *Phishing Email Response* solution pack by creating a demo phishing email alert. It then performs a set of actions to emulate the solution pack's behavior on receiving such alerts. |
 
 ## Playbook Collection
 
-|02 - Use Case - Phishing Email Response|
-| :- |
+| 02 - Use Case - Phishing Email Response |
+|:----------------------------------------|
 
-|Playbook Name|Description|
-| :- | :- |
-|Investigate Suspicious Email|Investigates an alert of type 'Suspicious Email'|
-|Email (Manual Upload) - Investigate|This playbook will extract email metadata from uploaded email file e.g. mail.eml or mail.msg|
-|Email (Manual Attach) - File to Alert (Suspicious Email)|This playbook allows to attach an email to alert of type suspicious email and investigates|
-|Email (Manual Upload) - Extract Attachments|This playbook will extract attachments, create indicators and link to parent alert|
-|Generate Phishing Email Alert|Generate a Phishing Email alert|
+| Playbook Name                                            | Description                                                           |
+|:---------------------------------------------------------|:----------------------------------------------------------------------|
+| Investigate Suspicious Email                             | Investigates alerts of type *Suspicious Email*                        |
+| Generate Phishing Email Alert                            | Generates a Phishing Email alert                                      |
+| Email (Manual Upload) - Investigate                      | Extracts email metadata from an uploaded email file                   |
+| Email (Manual Attach) - File to Alert (Suspicious Email) | Attaches emails to alerts of type *Suspicious Email* and investigates |
+| Email (Manual Upload) - Extract Attachments              | Extracts attachments, creates indicators, and links to parent alert   |

```

Comment:
youtube icon is not looking correct in below image, i.e. docs/res/phishing-email-response-thumbnail.png